RubberduckTests/Symbols/DeclarationFinderTests.cs

```diff
@@ -445,6 +446,44 @@ End Sub
             Assert.AreEqual(isConflict, conflicts.Where(cf => cf.IdentifierName.Equals(nameToCheck)).Any(), ConflictMessage(isConflict, nameToCheck, conflicts));
         }
 
+
+        //https://github.com/rubberduck-vba/Rubberduck/issues/4969
+        private const string projectOneModuleName = "projectOneModule";
+        private const string projectTwoModuleName = "projectTwoModule";
```

Comment:
not sure why these are outside of the test?

Reply:
Just wanted to avoid copying these strings within the test - and using parameterized testcases requires constants defined external to the testcase.

Reply:
I think using external constants here does not really buy anything, but also does not hurt. Now, you had to copy the variable names, which are used once each inside the test and inside the test case definition instead of the copying the literals, which are constants, too.